schedule/ext4_xen-pv.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name:           ext4_xen-pv
+description:    >
+  Test for ext4 filesystem in svirt-xen-pv.
+  Test modules 'grub_disable_timeout' and 'grub_test' in xen-pv are not scheduled
+  due to grub2 doesnt support xfb console.
+vars:
+  FILESYSTEM: ext4
+  FORMAT_DASD: install
+schedule:
+  - installation/bootloader_start
+  - installation/welcome
+  - installation/accept_license
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning_filesystem
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
+  - installation/installation_overview
```

Comment:
seems like you miss disable_grub_timeout

Reply:
[check this](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7510/files#diff-52242931c6b04699a9fbf06bb30d09d5R5)

> i am not sure if they are mandatory but there are two module missing compared to https://openqa.suse.de/tests/2897455

[check this](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7510/files#diff-52242931c6b04699a9fbf06bb30d09d5R5)